VContainer/Assets/VContainer/Runtime/ContainerBuilder.cs

```diff
@@ -68,7 +68,7 @@ public DiagnosticsCollector Diagnostics
             }
         }
 
-        readonly List<RegistrationBuilder> registrationBuilders = new List<RegistrationBuilder>();
+        internal readonly List<RegistrationBuilder> registrationBuilders = new List<RegistrationBuilder>();
```

Comment:
To get registered types from scope after calling `Configure` method I had to make that field internal.
See: LinkGeneratorTypeHelper.GetTypesFromLifetimeScopesInAsseblies:99